real_estate/models/estate_property.py

```diff
@@ -0,0 +1,39 @@
+from odoo import fields, models
+import datetime
+
+class RealEstateProperty(models.Model):
```

Comment:
```suggestion
class EstateProperty(models.Model):
```
Make sure your file name, class Name and `_name` must be similiar